bpf/bpf_lxc.c

```diff
@@ -498,9 +498,12 @@ int tail_handle_ipv6(struct __ctx_buff *ctx)
 					CTX_ACT_DROP, METRIC_EGRESS);
 
 #ifdef ENABLE_CUSTOM_CALLS
-	if (!encode_custom_prog_meta(ctx, ret, dstID))
+	if (!encode_custom_prog_meta(ctx, ret, dstID)) {
```

Comment:
Check patch doesn't want camel case in the C code. I guess you have to find all references to "dstID" and refactor them :man_shrugging: .
```suggestion
	if (!encode_custom_prog_meta(ctx, ret, dst_id)) {
```

Reply:
These warnings are fine to ignore. It can't be on the author of a pull request to fix all of those.

Reply:
Agreed, these kind of false positives are one reason why we never marked checkpatch as `required` to merge the PR. I don't mind changing the `dstID` as a follow-up, although I don't really see the point other than pleasing checkpatch. But that would be for a separate PR anyway.

Re-asking for a review.